libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -1234,6 +1388,69 @@ pub fn find_complete_timing_marks_from_partial_timing_marks(
         );
     }
 
+    const MIN_REQUIRED_CORNER_MARK_SCORE: f32 = 0.5;
+    let mut missing_corners = vec![];
+
+    if partial_timing_marks.top_left_rect.is_none() {
```

Comment:
I wonder if we should score every left and right inferred timing mark to give us more confidence.

Reply:
Maybe. I wonder what we'd do if any didn't meet the threshold. Maybe reject if there's more than a handful?

Reply:
I was thinking fail if any don't meet the threshold. I feel like every side timing mark should easily meet it, unless they've been ripped out or colored in-between. Protecting against ripping out a chunk of the ballot seems useful, though maybe not the most realistic worry.

Reply:
I'd like to consider this as a future enhancement. I think it could be a good idea, but I'd want to do more testing. cc @arsalansufi